src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -118,6 +118,32 @@ public static byte[] FixupKeyParity(this byte[] key)
             }
             return oddParityKey;
         }
+
+        public static void ConvertIntToByteArray(uint dwInput, ref byte[] counter)
```

Comment:
We can't rename parameters, that's a breaking change. http://index/?query=ConvertIntToByteArray&rightProject=mscorlib&file=system%5Csecurity%5Ccryptography%5Cutils.cs&line=668